x-pack/plugins/lens/public/state_management/init_middleware/load_initial.ts

```diff
@@ -341,7 +341,9 @@ export function loadInitial(
                       !(initialInput as LensByReferenceInput)?.savedObjectId &&
                       currentSessionId
                         ? currentSessionId
-                        : data.search.session.start(),
+                        : !inlineEditing
+                        ? data.search.session.start()
+                        : undefined,
```

Comment:
Why do we want to avoid a search session during inline editing?

Reply:
It doesnt avoid it, it just uses the existing. This is solving a bug when you try to save a lens embeddable which was edited via the flyout.

Reply:
> when the app doesnt use search sessions

Huh, surprising that this causes a bug—I thought search sessions are just useful for triggering a refresh.

But since this fixes a bug, maybe we could just add a comment about it?